### PR TITLE
Fix: focus order

### LINKED ISSUE #750: Focus order is not in logical manner after "Close" button on Graph-Explorer page.
**Describe the bug**
Users who rely on keyboards will affect to navigate, if focus order is not in logical manner after  "Close" button on Graph-Explorer page.

**To Reproduce**
Step 1: Open the Graph Explorer URL in Edge browser.
Step 2: Navigate to "Close" button on Graph-Explorer page..
Step 3: Now press Tab key and Observe whether focus indicator is in logical manner.

Focus order is not in logical manner after "Close" button .
After "Close" button, focus is moving to “Share query” button instead of moving “Response preview” button.

**Expected behavior**
Focus order should be in logical manner after "Close" button .
After "Close" button, focus should move to “Response preview” button instead of moving  “Share query” button.

**Desktop (please complete the following information):**
OS: OS build Version 2004 (OS Build 20170.1000)
URL: Graph Explorer - Microsoft Graph
Browser: Anaheim Edge Dev Version: 87.0.634.0 (Official build) dev (64-bit)

### PR DESCRIPTION
**Pull request by ElinorW**
## Overview

Fixed the focus order after "Close" button by including the 'expand' and 'share' links as pivot items.
After "Close" button, focus will move to “Response preview” button instead of moving  “Share query” button.

Fixes #750 

### Demo

Optional. Screenshots, `curl` examples, etc.

### Notes

Optional. Ancillary topics, caveats, alternative strategies that didn't work out, anything else.

## Testing Instructions

Step 1: Open the above Graph Explorer URL in Edge browser.
Step 2: Navigate to "Close" button on Graph-Explorer page..
Step 3: Now press Tab key and Observe whether focus indicator moves to Response Preview

### CODE REVIEW

File: src/app/views/query-response/QueryResponse.tsx [hunk 1]
```diff
@@ -180,6 +172,15 @@ class QueryResponse extends Component<IQueryResponseProps, IQueryResponseState>
...
   }
+
+  private showResponse(pivotItems: JSX.Element[]) {
+    return <Pivot className='pivot-response'
+      styles={{ root: { display: 'flex', flexWrap: 'wrap' } }}
+      onLinkClick={(pivotItem) => onPivotItemClick(pivotItem)}
+    >
+      {pivotItems}
+    </Pivot>;
+  }
```
Comment (thewahome) [inline src/app/views/query-response/QueryResponse.tsx:183]: This function is not in use

**Review by thewahome — CHANGES_REQUESTED** [review]

File: src/app/views/query-response/QueryResponse.tsx [hunk 2]
```diff
@@ -48,10 +52,29 @@ class QueryResponse extends Component<IQueryResponseProps, IQueryResponseState>
...
+    if (key && key.includes('share')) {
+      this.handleShareQuery();
+    }
+  }
+
+  public renderItemLink(link: any) {
+    return (
+      <TooltipHost content={link.title} id={getId()} calloutProps={{ gapSpace: 0 }} >
+        <Icon iconName={link.itemIcon} style={{ paddingRight: 5 }} />
+        {link.headerText}
```
Comment (thewahome) [inline src/app/views/query-response/QueryResponse.tsx:73]: Here we should show the translated text instead

Comment (ElinorW) [reply src/app/views/query-response/QueryResponse.tsx:73]: where? at the `{link.headertext}` ??

Comment (ElinorW) [reply src/app/views/query-response/QueryResponse.tsx:73]: Nvm I get what you meant

**Review by thewahome — APPROVED** [review]

**Pull request merged**